init/kali/waagent-apt.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Microsoft Azure Linux Agent (apt setup)
+#Requires=waagent-provision.service
+#After=waagent-provision.service
+Before=multi-user.target
+ConditionVirtualization=microsoft
+
+[Service]
+ExecStart=/usr/share/waagent/apt-setup
+StandardOutput=journal+console
+
+[Install]
+WantedBy=walinuxagent.service
```

Comment:
Will this ensure that the `apt-setup` script is called before `walinuxagent.service` is run?